src/main/kotlin/com/amazon/ion/plugin/intellij/IonIcons.kt

```diff
@@ -0,0 +1,20 @@
+package com.amazon.ion.plugin.intellij
+
+import com.intellij.openapi.util.IconLoader
+
+/**
+ * Common Icons used throughout the plugin.
+ */
+object IonIcons {
+    /**
+     * Alternative file type logos:
+     * - Old nucleus looking logo - ion-file-type.ping
```

Comment:
Was this meant to be `.png` instead of `.ping`?

I wish GitHub would display all of the new image assets. 😞

Reply:
Yes I can fix that comment.